content/2.0.html.haml

```diff
@@ -90,109 +89,121 @@ title: "Jenkins 2.0"
+
+  
+%h3 Solution
+
+%p For new users, Jenkins 2.0 starts off with set of recommended plugins to help get you started with the right set of tools to get up and running with Jenkins quickly.
+
+%p There are a number of useful improvements in user interface, such as an improved configuration page, to make users lives easier.
+
+
+%h4 Key Features and Benefits for Pipelines
+
+%table{:border => 1} 
+  %tr
+    %th Features
+    %th Benefits
+  %tr 
+    %td New getting started experience
+    %td
+      %ul
+        %li Start with a set of recommended plugins that match a users preferred use case. 
+        %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
+          %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
           :width => 550}/
 
-    %p
-      Learn more about this on our
-      %a{:href => '/solutions/pipeline'}
-        Pipeline solutions page
-      !
-
-  %dt
-    Out of the box: Easy to get it right
-  %dd
-    %p
-      For new users, Jenkins 2.0 starts off with set of recommended plugins to
-      help get you started with the right set of tools to get up and running with
-      Jenkins quickly.
-
-      For the more adventurous users, the Jenkins 2.0 initial setup process
-      also allows you to pick and choose exactly the plugins you want to meet
-      your specific needs. The list of plugins is hand-picked by Jenkins experts
-      from our popular plugins to ensure the best getting-started experience for
-      new users.
-
-    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
-      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
-          :width => 550}/
+  %tr
+    %td New Configurations tab view
+    %td 
+      %ul
+        %li Easily configure jobs with scoped configuration. You no longer need to go through the entire set of configuration choices to change a few values
 
-    %p
-      Additionally,
-      %em
-        starting in an upcoming alpha release
-      , the initial setup will be protected so that your Jenkins is always safe from unauthorized access.
 
+%h2 Backward Compatible
 
-    %p
-      Of course, users will always be able to add plugins after the initial
-      setup through the current "Manage Plugins" interface.
+%h3 Problem
 
-  %dt
-    Improved user interface
-  %dd
-    A number of small, but useful improvements, have been made to tidy up the
-    interfaces you use every day.
+%p Jenkins is mission critical for many teams. The new major release has valuable enhancements, yet if the upgrade process is time-consuming or risky many users will not upgrade.  Leading to the question: In my experience major version upgrade "break things"!, is Jenkins 2.0 backwards compatible?
```

Comment:
The question is long for what it's actually about. It's not like the concept of backward compatibility is unknown to the audience. Also, this confuses problems upgrading with backwards compatibility.